simplified-books-api/src/main/java/org/nypl/simplified/books/api/BookmarkJSON.kt

```diff
@@ -60,10 +60,26 @@ object BookmarkJSON {
     node: ObjectNode
   ): Bookmark {
 
+    // Older bookmarks store chapter progress in a top-level property, instead of inside
+    // location.progress.
+
+    val chapterProgress = JSONParserUtilities.getDouble(node, "chapterProgress")
+
+    val deserializedLocation = BookLocationJSON.deserializeFromJSON(
+      objectMapper, JSONParserUtilities.getObject(node, "location"))
+
+    val location =
+      if (deserializedLocation.progress == null && chapterProgress != null)
```

Comment:
Kotlin coding conventions require braces on any multiline if / else statements.

> If the condition of an if or when statement is multiline, always use curly braces around the body of the statement.

If the expression is short enough to fit on a single line you can omit the braces.

https://kotlinlang.org/docs/reference/coding-conventions.html#formatting-control-flow-statements

Honestly not sure why ktlint didn't flag this.

Reply:
Looks like there's an experimental rule to enforce this.

https://github.com/pinterest/ktlint/blob/c7bdae030dc55c/ktlint-ruleset-experimental/src/main/kotlin/com/pinterest/ktlint/ruleset/experimental/MultiLineIfElseRule.kt

Reply:
Thanks, fixed.